bot/code_review_bot/workflow.py

```diff
@@ -436,6 +436,8 @@ def build_task(self, task_status):
             return ExternalTidyTask(task_id, task_status)
         elif name == "source-test-taskgraph-diff":
             return TaskGraphDiffTask(task_id, task_status)
+        elif name.startswith("source-test-shadow-scheduler"):
```

Comment:
If we do this, we have to maintain a list of tasks that we don't care about, I think it's preferable to just return None in the last `else`, since if the artifact is not available we will never be able to parse it in DefaultTask anyway.